### PR TITLE
Get week year method

### PR DESCRIPTION
**Pull request by Ameya02**
### Description
- Added the calender.getWeekYear() method for Java
- Closes issue #3100 

### Type of Change

- Adding a new entry
### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+Title: '.getWeekYear()'
+Description: 'Returns the week year'
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md:3]:
```suggestion
Description: 'Returns the week year.'
```

File: content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.getWeekYear()`** method of the `Calendar` class returns the time represented by a `Calendar` object in the form of week year.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md:15]:
```suggestion
The **`.getWeekYear()`** method of the `Calendar` class returns the time represented by a `Calendar` object in the form of the week year.
```

File: content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```pseudo
+calendar.getWeekYear()
+```
+
+Returns the time represented by this `Calendar` object in week year as a `integer`.
+
+## Example
+
+This example code shows the basic use of the getWeekYear() method
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getWeekYear/getWeekYear.md:27]:
```suggestion
This example code shows the basic use of the `getWeekYear()` method:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Ameya02 I did one edit to fix the changes to the `yarn.lock` file. Then on review I found a few things to address.

**Review by SSwiniarski — APPROVED** [review]
@Ameya02 Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@Ameya02 @SSwiniarski Just made a couple of minor edits, looks good to merge

**Pull request merged**